src/message.lisp

```diff
@@ -117,20 +121,17 @@
 ;;              `(:reply_to_message_id ,reply-to-message-id)))))
 ;;     (make-request bot "sendMessage" options)))
 
+(defun forward-message (bot chat from-chat message &key disable-notification)
+  "https://core.telegram.org/bots/api#forwardmessage"
+  (let ((options
+          `(:|chat_id| ,(get-chat-id chat)
```

Comment:
Please, use a LIST instead of backticks here and below.

Here is a demonstration why backticks are dangerous in a context like this: https://twitter.com/svetlyak40wt/status/1535769530672324608

Reply:
Done!